connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SessionHolder.scala

```diff
@@ -179,12 +179,13 @@ case class SessionHolder(userId: String, sessionId: String, session: SparkSessio
    */
   private[service] def interruptAll(): Seq[String] = {
     val interruptedIds = new mutable.ArrayBuffer[String]()
+    val operationsIds = SparkConnectService.streamingSessionManager.cleanupRunningQueries(this)
     executions.asScala.values.foreach { execute =>
       if (execute.interrupt()) {
         interruptedIds += execute.operationId
       }
     }
-    interruptedIds.toSeq
+    interruptedIds.toSeq ++ operationsIds
```

Comment:
This is a little bit weird one. The operations IDs are not from actual running `executions` within `SessionHolder` but the operation ID from the past execution (that launched a streaming query).

I am not sure if it makes sense, or if we should do something else, @juliuszsompolski @WweiL

Reply:
Yea this indeed is a bit weird. 

What are operation IDs used normally? For a batch query do we send them back to client so users can use that to identify ongoing operations?

Reply:
Also in classic spark is there such concept for batch queries?

In streaming a query is identified by (id, runId).

But we can also ask the user to keep a client side operation Id <-> streaming query map so it can be workaround